n/numpy/build_info.json

```diff
@@ -1,25 +1,13 @@
 {
-  "maintainer": "shubham-garud",
-  "package_name": "numpy",
-  "github_url": "https://github.com/numpy/numpy.git",
-  "version": "v2.2.2",
-  "wheel_build" : true,
-  "default_branch": "main",
-  "package_dir": "n/numpy",
-  "build_script": "numpy_v2.2.2_ubi_9.3.sh",
-  "docker_build": false,
-  "validate_build_script": true,
-  "use_non_root_user": false,
-  "v1.26.0": {
-    "build_script": "numpy_1.26.0_ubi_9.3.sh"
-  },
```

Comment:
Version 1.24.2 can use the numpy_1.23.5_ubi_9.3.sh script to build successfully.
Other changes are not necessary.

```
"v1.24.2": {
  "build_script": "numpy_1.23.5_ubi_9.3.sh"
},
```